src/test/java/io/appium/java_client/android/UIAutomator2Test.java

```diff
@@ -29,9 +35,20 @@
     /**
      * initialization.
      */
-    @BeforeClass public static void beforeClass() {
-        service = AppiumDriverLocalService.buildDefaultService();
+    @BeforeClass public static void beforeClass() throws MalformedURLException, UnknownHostException {
+        AppiumServiceBuilder appiumServiceBuilder = new AppiumServiceBuilder()
+                .usingPort(4357);
+        service = appiumServiceBuilder.build();
         service.start();
+        InetAddress ipadd =InetAddress.getLocalHost();
+        String n=ipadd.toString();
+        String ip = n.substring(n.indexOf("/")+1);
+
+        System.out.println("-------------------------------------------");
+        System.out.println("IP address :"+n.substring(n.indexOf("/")+1));
```

Comment:
Yeah otherwise `service` doesn't get the URL in travis env. Have to investigate that why itsn't getting it. e.g.https://travis-ci.org/SrinivasanTarget/java-client/jobs/391881959

Reply:
I mean you've already assigned that value to `ip` above

Reply:
Oh Ok